CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ endif()
 # Example 'v1.0.0'. They MUST not contain a pre-release label such as '-beta'.
 set(FIRMWARE_VERSION "v4.3.0")
 set(FIRMWARE_BTC_ONLY_VERSION "v4.3.0")
+set(FIRMWARE_BITBOXBASE_VERSION "v1.0.0")
```

Comment:
4.3.0 please - there is a lot of api stuff in both clients that rely on this.

Reply:
ack